src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,15 @@
+import { NavBar as Nav } from 'antd-mobile'
+import styled from 'styled-components'
+
+const Top = styled.div`
```

Comment:
Can you take a look at the semantic components to see if we don't have anything more appropriate than a div for this case?